pkg/cmd/release/download/download.go

```diff
@@ -223,6 +223,7 @@ func downloadAssets(dest *destinationWriter, httpClient *http.Client, toDownload
 	for w := 1; w <= numWorkers; w++ {
 		go func() {
 			for a := range jobs {
+				io.StartProgressIndicatorWithLabel(fmt.Sprintf("Downloading %s", a.Name))
```

Comment:
What do you think about adding in a call at the end of this function to `io.StopProgressIndicator()`? I don't necessarily like how this code relies on the calling function to stop the progress indicator. Right now it is only being invoked from one place but that is not a guarantee for the future.

Reply:
I think that makes sense. I added a call to `io.StopProgressIndicator()` after each asset is downloaded, let me know if you had something different in mind